### PR TITLE
Fix Valhalla value types opcode remap typos in ClassFileOracle.cpp

### PR DESCRIPTION
**Pull request by eric98zhang**
Signed-off-by: Eric Zhang <eric99.zhang@gmail.com>

### CODE REVIEW

File: runtime/bcutil/ClassFileOracle.cpp [hunk 1]
```diff
@@ -1392,13 +1392,13 @@ ClassFileOracle::walkMethodCodeAttributeCode(U_16 methodIndex)
 	U_8 *code = codeAttribute->code;
 	for (U_32 codeIndex = 0; codeIndex < codeAttribute->codeLength; codeIndex += step) { /* NOTE codeIndex is modified below for CFR_BC_tableswitch and CFR_BC_lookupswitch */
 
-#if defined(J9VM_OPT_VALHALLA_VALUETYPES)
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
```
Comment (tajila) [inline runtime/bcutil/ClassFileOracle.cpp]: please add a TODO comment saying something like:  The current edition of LWVT currently maps the ValueType bytecodes to opcodes that are used internally. This is a work around reads in the LWVT bytecodes and remaps them to opcodes that do not conflict. Once, there is an official spec that specifies which opcodes the new LWVT bytecodes will have, we will remap the internal bytecodes so they do not conflict.

File: runtime/bcutil/ClassFileOracle.cpp [hunk 2]
```diff
@@ -1392,13 +1392,13 @@ ClassFileOracle::walkMethodCodeAttributeCode(U_16 methodIndex)
 	U_8 *code = codeAttribute->code;
 	for (U_32 codeIndex = 0; codeIndex < codeAttribute->codeLength; codeIndex += step) { /* NOTE codeIndex is modified below for CFR_BC_tableswitch and CFR_BC_lookupswitch */
 
-#if defined(J9VM_OPT_VALHALLA_VALUETYPES)
```
[the remark this answers is not in the document]
Comment (eric98zhang) [reply runtime/bcutil/ClassFileOracle.cpp]: yes! http://vmfarm.ottawa.ibm.com/build_info.php?build_id=379248

File: runtime/bcutil/ClassFileOracle.cpp (hunk 1 above)
Comment (eric98zhang) [reply runtime/bcutil/ClassFileOracle.cpp]: new PR: https://github.com/eclipse/openj9/pull/1234

**Comment by pshipton** [comment]
Apologies for merging this early.

**Pull request merged**